spec.md

```diff
@@ -180,6 +180,7 @@ The `Docker-Content-Digest` header, if present on the response, returns the cano
 If the digest does differ, it MAY be the case that the hashing algorithms used do not match.
 See [Content Digests](https://github.com/opencontainers/image-spec/blob/v1.0.1/descriptor.md#digests) <sup>[apdx-3](#appendix)</sup> for information on how to detect the hashing algorithm in use.
 Most clients MAY ignore the value, but if it is used, the client MUST verify the value against the uploaded blob data.
+If the `<reference>` part of a manifest request is a digest, clients SHOULD additionally verify that the response body matches this digest.
```

Comment:
We could change "response body" to instead mention it as the manifest digest. There are deprecated but still in use manifest formats which have an envelope in which the digest is checked.  It is possible for future formats to put the manifest in an envelope or require decoding/decrypting that doesn't match the response body.

Reply:
While we're at it, the line above calls this "uploaded blob data". Should we replace both with just the term "manifest" and leave it to the reader to understand what part of the response that is?

Reply:
For blobs, I think we are fine calling it the content body digest. It's only for manifests where there are some possible differences, since registries process the manifest content, and the digest could be for the envelope data rather than the full body of the response.

Reply:
```suggestion
If the `<reference>` part of a manifest request is a digest, clients SHOULD verify the returned manifest matches this digest.
```